types/country-telephone-data/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for country-telephone-data 0.6
+// Project: https://github.com/mukeshsoni/country-telephone-data#readme
+// Definitions by: Kyle Hensel <https://github.com/k-yle>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface Country {
+    name: string;
+    /** The {@link https://w.wiki/4kP |ISO 3166-1 alpha-2} country code (in **lowercase**) */
+    iso2: string;
+    dialCode: string;
+    /** A format string for phone numbers. **This could be an empty string.** */
+    format: string;
+    hasAreaCodes: boolean;
```

Comment:
```suggestion
    hasAreaCodes?: true;
```
it's seems to be either true or falsy:
https://github.com/mukeshsoni/country-telephone-data/blob/master/country_telephone_data.js#L360

Reply:
thanks, changed